src/NServiceBus.Transport.RabbitMQ.CommandLine/NServiceBus.Transport.RabbitMQ.CommandLine.csproj

```
@@ -37,4 +37,8 @@
     <InternalsVisibleTo Include="NServiceBus.Transport.RabbitMQ.CommandLine.Tests" Key="$(NServiceBusTestsKey)" />
   </ItemGroup>
 
+  <ItemGroup Label="Pinned transitive dependencies">
+    <PackageReference Include="System.Drawing.Common" Version="8.0.4" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="8.0.0" />
+  </ItemGroup>
```

For readability, keep a double newline between sections
```suggestion
  </ItemGroup>

```